sentences/fr/_common.yaml

```diff
@@ -587,6 +588,8 @@ expansion_rules:
   atil: "(ont|a)[-][ ][t][ ][-][(il[s]|elle[s])]"
   quel: "quel[le][s]"
   quelest: "<quel> (est|sont)"
+  donnemoi: "(donne[s]|dis)[(-| )moi]"
+  what_is_the_class_of_name: "<quelest> (le |la |l'|les )<class> [(indiqué[e][s]|mesuré[e][s]|renvoyé[e][s]|restant[e][s]|retourné[e][s]|utilisé[e][s]|produit[e][s]|consommé[e][s]|donné[e][s]) ][(par|<dans>|sur)] [<le>]{name} [<dans> [<le>]{area}]"
```

Comment:
```suggestion

```

Reply:
Let's remove this one, it's unused anymore.
And then let's finally merge this pr ;) 

Thx a lot man!